google/resource_dataproc_cluster_test.go

```diff
@@ -107,6 +107,9 @@ func TestAccDataprocCluster_basic(t *testing.T) {
 					// Default behaviour is for Dataproc to autogen or autodiscover a config bucket
 					resource.TestCheckResourceAttrSet("google_dataproc_cluster.basic", "cluster_config.0.bucket"),
 
+					// Default behavior is for Dataproc to not use only internal IP addresses
+					resource.TestCheckResourceAttr("google_dataproc_cluster.basic", "cluster_config.gce_cluster_config.internal_ip_only", "false"),
```

Comment:
This should be `cluster_config.0.gce_cluster_config.0.internal_ip_only` because of the way nested resources are stored in the Terraform state

Reply:
Fixed in 365df40!